templates/api.hbs

```diff
@@ -32,7 +54,9 @@ export default function(app: Express, impl: t.{{className name}}Api) {
 						{{#unless schema}}
 						return undefined
 						{{else}}
-						{{#ifmatch mediaType.mimeType '/json'}}
+						{{#ifmatch mediaType.mimeType 'multipart/form-data'}}
```

Comment:
I'd like to use `{{#if (isContentMultipart .)}` here instead of `{{#ifmatch mediaType.mimeType 'multipart/form-data'}}`